config/manifests/bases/cluster-logging.clusterserviceversion.yaml

```diff
@@ -87,15 +87,13 @@ metadata:
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-logging
     operators.openshift.io/infrastructure-features: '["disconnected","proxy-aware"]'
-    operators.operatorframework.io/builder: operator-sdk-unknown
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
     support: AOS Logging
     target.workload.openshift.io/management: '{"effect": "PreferredDuringScheduling"}'
   labels:
     operatorframework.io/arch.amd64: supported
     operatorframework.io/arch.ppc64le: supported
     operatorframework.io/arch.s390x: supported
-  name: clusterlogging.v0.0.0
+  name: cluster-logging.v0.0.0
```

Comment:
This may break upgrade without explicitly using "replace". My work with MOLO to modernize the deployment led me to:
* package name must be consistent for one operator to supersede another; the package name can not vary by operator name can
* the one operator (new) must explicitly replace the other (old) via replaces in the CSV)

The last point would allow us to change from "clusterlogging" to "cluster-logging".  Replaces is good practice (I believe) but we don't use it to date

Reply:
I'll go back to the way it was. We may want to make the hyphenation more consistent in future, but its best not to mix it in with this commit.